bin/varnishd/VSC_main.vsc

```diff
@@ -875,6 +875,12 @@
 	:oneliner:	HCB Inserts
 
 
+.. varnish_vsc:: esi_req
+	:group: wrk
+	:oneliner:	ESI subrequests
+
+	Number of ESI subrequests made.
+
```

Comment:
would this more naturally placed below `client_req`?

Reply:
Agreed.

Reply:
Moved